src/app/garfile/arrival/index.njk

```diff
@@ -184,5 +194,6 @@
       selectElement: document.querySelector('#arrivalPort'),
       minLength: 2,
     });
+    document.getElementById('arrivalPort').setAttribute('aria-describedby', 'arrivalPort-hint{{ ' arrival-error' if errors | containsError('arrivalPort') }}');
```

Comment:
Javascript here to add the aria-describedby attribute to the accessible autocomplete. This is something that probably the component itself should have done anyway...